app/scripts/sentry-install.js

```diff
@@ -14,13 +14,13 @@ global.sentry = setupSentry({
  * toggleSession method added to sentry in setupSentry to start automatic
  * session tracking.
  */
-function waitForStateHooks() {
+async function waitForStateHooks() {
   if (global.stateHooks.getSentryState) {
     // sentry is not defined in dev mode, so if sentry doesn't exist at this
     // point it means that we are in dev mode and do not need to toggle the
     // session. Using optional chaining is sufficient to prevent the error in
     // development.
-    global.sentry?.toggleSession();
+    await global.sentry?.toggleSession();
```

Comment:
@brad-decker do you recall why "toggle" was used here rather than "start"?

Now that this is async, it occurred to me that this might introduce a race condition if the session is started elsewhere but toggled here, if the "start" call executes first. I don't see "start" being called anywhere today upon load, but still it might be nice to avoid this footgun if we can.

Reply:
@Gudahtt there is no reason other then I made an assumption that state assessed in the toggle would be the same as it would be here, but you are correct. We can remove toggle altogether and just do the state comparison here.

Reply:
Sorry I was mobile when I answered this question which was a disservice to you and myself. We call toggle here because it contains the logic of checking for both the current state in the state tree as well as the existence of a session -- but you are correct the default state is no session and autoSessionTracking off. It should be sufficient here that if metaMetrics is enabled to just call the start method.

My original response was in relation to the actions.ts usage of toggleSession which I mistakenly thought was the only one (just from memory) while I was out of pocket.

Reply:
Thanks! I've adjusted this to call `startSession` instead in a086ddee33, let me know if that makes sense.

I didn't add any state check, but that already happens internally in `startSession`

Reply:
Yep that makes sense to me